Libraries/LibWeb/HTML/HTMLTitleElement.cpp

```diff
@@ -50,4 +58,28 @@ void HTMLTitleElement::set_text(String const& value)
     string_replace_all(value);
 }
 
+void HTMLTitleElement::consider_propagate_title_update()
+{
```

Comment:
You've now removed the empty title logic altogether, which means that technically we could still send an erroneous empty title update to the chrome. My idea was that if the title is empty, you start the timer and return here.

That way, if the empty title update was intentional (instead of being 1 of 2 updates), the update will take at most 5ms.